tests/foreman/cli/test_errata.py

```diff
@@ -1091,22 +1092,28 @@ def test_positive_list_filter_by_product_id_and_org_label(self):
             set([])
         )
 
-    @stubbed()
-    def test_positive_list_filter_by_product_name(self):
```

Comment:
so it is not possible to filter errata by product name and it is expected by application design, right?